lineapy/utils/deprecation_utils.py

```diff
@@ -79,21 +82,17 @@ def _pad_whitespace(source):
 
 
 def get_source_segment(source, node, padded=False):
-    """Get source code segment of the *source* that generated *node*.
-
-
-    .. note::
-
-        This is a polyfill for the ast.get_source_segment function
-        that was introduced in python 3.8.
-
-        If some location information (`lineno`, `end_lineno`, `col_offset`,
-        or `end_col_offset`) is missing, return None.
+    """
+    Get source code segment of the *source* that generated *node*.
 
-        If *padded* is `True`, the first line of a multi-line statement will
-        be padded with spaces to match its original position.
+    This is a polyfill for the ast.get_source_segment function
```

Comment:
Use `??? note` to make it compatible with earlier documentation or make it a comment

Reply:
Fixed